Apps/phokta/okta.json

```diff
@@ -15,6 +15,99 @@
             "type": "test",
             "verbose": ""
         },
+        {
+            "action": "send push notification",
+            "description": "Validate the user prompt with push notification",
+            "read_only": true,
```

Comment:
false since this action posts

Reply:
This action does not **modify/write** on user's profile.

Though, it uses `POST` http method per Okta API documentation (requesting for verification).   

Let me know either way!

Reply:
I think, to be safe, we should set this to false.